documentation/src/main/markdown/packagemgrs/cpan.md

```diff
@@ -0,0 +1,18 @@
+# CPAN Support
+
+## Related properties
+
+[Detector properties](../properties/detectors/cpan.md)
+
+## Overview
+
+The CPAN detectable will run if it finds a Makefile.PL file.
```

Comment:
detectable or Detector?

Reply:
Thanks. I now went with "detector". Lowercase because that's what we seem to have in a few other similar pages.